### PR TITLE
docker: Add an html title to the run container btn

### PR DESCRIPTION
**Pull request by moolitayer**

### CODE REVIEW

**Comment by moolitayer** [comment]
screenshot from 2015-09-19 04-12-45

File: pkg/docker/overview.js [hunk 1]
```diff
@@ -179,6 +179,7 @@ define([
             var added = false;
             if (!tr.length) {
                 var button = $('<button class="btn btn-default btn-control fa fa-play">').
+                    attr("title", "run image").
```
Comment (stefwalter) [inline pkg/docker/overview.js]: Thanks. Could you capitalize the 'R' and also wrap this in a ```_()``` call like this: ```_("Run image")``` ... which makes this string translatable.

**Comment by moolitayer** [comment]
address comment and update screenshot

**Comment by moolitayer** [comment]
fyi @stefwalter

**Pull request closed without merging**